lib/election.rb

```diff
@@ -1,2 +1,24 @@
 class Election
+  attr_reader :year, :races, :name_votes_hash
+  
+  def initialize(year)
+    @year = year
+    @races = []
+  end
+
+  def add_race(race)
+    @races << race
+  end
+
+  def candidates 
+    @races[0].candidates
```

Comment:
This will only get you the candidates for the first race, but we want *all* of the races. Consider this option:

```
all_candidates = []
@races.each do |race|
    race.candidates.each do |candidate|
        all_candidates << candidate
    end
end
```

You could also use flat_map to map each race to their candidates
```